src/resources/uart_esp32.cc

```diff
@@ -319,6 +366,9 @@ UART_ISR_INLINE uint8* TxBuffer::read(uword *received, uword max_length) {
   uword maximum_to_receive = Utils::min(max_length, static_cast<uword>(transfer_header_.remaining_data_length_));
 
   if (maximum_to_receive == 0) {
+    // TODO(kasper): What does this mean? The payload is empty? Do
+    // we need to handle this as a case where the TxBuffer is empty?
```

Comment:
This will be the case when there is no data but there is a break length.

Reply:
So maybe we don't need the call to is_empty_isr? If I can add a parameter to handle_empty_isr that would be pretty nice.